mongodb-atlas-sql-interface.md

```diff
@@ -5,70 +5,102 @@
+     Use the name of the federated database obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).
+   - The logging level for the underlying ODBC driver.  
+   If you need more information in order to troubleshoot, you can increase the logging level. Please note that increasing the logging level will make the driver slower. Only use the DEBUG level while troubleshooting issues.
+
    
-   Select **Continue**. 
-6. Enter your Atlas MongoDB Database access username and password and select **Connect**.
-> [!NOTE]
-> Once you enter your username and password for a particular Atlas federated database, Power BI Desktop uses those same credentials in subsequent connection attempts. You can modify those credentials by going to **File > Options and settings > Data source settings**. 
+   Select **OK**.  
 
-7. In **Navigator**, select one or multiple elements to import and use in Power BI Desktop. 
+   ![Atlas SQL connection dialogue](./media/mongodb/mongodb_connection_dialogue.png)
+   
+5. Enter your Atlas MongoDB Database access username and password and select **Connect**.  
+
+   ![Authentication](./media/mongodb/mongodb_authentication.png)  
+
+   > [!NOTE]
+   > Once you enter your username and password for a particular Atlas federated database, Power BI Desktop uses those same credentials in subsequent connection attempts. You can modify those credentials by going to **File** > **Options and settings** > **Data source settings**.  
+
+6. In **Navigator**, select one or multiple elements to import and use in Power BI Desktop. 
    Then select either **Load** to load the table in Power BI Desktop, or **Transform Data** to open the Power Query 
-   Editor where you can filter and refine the set of data you want to use, and then load that refined set of data into 
+   editor where you can filter and refine the set of data you want to use, and then load that refined set of data into 
    Power BI Desktop.
 
+## Connect to MongoDB Atlas federated database using Atlas SQL interface from Power Query Online
+
+To connect using the Atlas SQL interface:
+
+1. Select **MongoDB Atlas SQL** from the **Power Query - Choose data source** page.
+2. On the **Connection settings** page, fill in the following values:
+    - The **MongoDB URI**. _Required_.   
+      Use the MongoDB URI obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).  Make sure that it doesn't contain your username and password. URIs containing username and/or passwords are rejected.
+    - Your federated **Database** name. _Required_  
+      Use the name of the federated database obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).
+    - Enter a **Connection name**.
+    - Choose a **Data gateway**.
+    - Enter your Atlas MongoDB Database access username and password and select **Next**.
+
```

Comment:
Log level information should also be here.  I'm pretty sure the Power BI desktop connection dialogue will match what is in the Online connection settings.